generators/server/templates/src/main/java/package/web/rest/UserResource.java.ejs

```diff
@@ -149,6 +153,7 @@ public class UserResource {
         <%_ if (authenticationType !== 'oauth2') { _%>
         this.userRepository = userRepository;
         this.mailService = mailService;
+        this.forbiddenOrderedProperties = Collections.unmodifiableList(List.of("resetKey", "password"));
```

Comment:
`List.of()` is only available since Java 9. I thought it was decided to stay with Java 8 syntax.
Also, `List.of()` already returns an immutable collection, there's no need for the additionnal `Collections.unmodifiableList`

Reply:
You're right, I'll revert it to Java 8 syntax.